### DISCUSSION TITLE
Audio Equalizer Option enabled by Default

### DISCUSSION THREAD

**Discussion by Raza-0007** [smplayer-dev/smplayer#954] [General]
@smplayer-dev,

Ricardo, I just noticed that on a fresh out-of-the box smplayer, the audio equalizer is enabled by default. I don't know if this is a recent change, or has been there for a while, because I always port over my old configuration to the new smplayer, so my audio equalizer is always disabled.

Most media players like VLC have the audio equalizer disabled by default, and the reasoning is that if someone has set a system wide audio equalizer through their sound card, then this option will cause the sound to pass through two separate equalizers, which is not a good thing. 

In smplayer, the audio equalizer is enabled by default with a "Flat Preset", which actually clips the high bass and high treble of the media, giving a bland output. You can immediately notice it if you are used to listening to your media with it disabled. A person new to smplayer will just assume that smplayer sounds bad.

I recommend that this setting should be disabled by default, and left to the user if they wish to enable it. What do you think?

~Raza

**Comment by smplayer-dev** [comment]
I didn't notice anything strange with the audio, but anyway I've just disabled it.
https://github.com/smplayer-dev/smplayer/commit/78b1f4498027d7402530410e4a19c5c8f166ea75

**Reply by Raza-0007** [reply]
Thanks.... The audio does not sound strange, it just sounds flat. You notice it more during music playback. Play a music file with heavy bass, with the equalizer enabled with a flat preset, smplayer just sounds....well flat. With the equalizer disabled you suddenly hear the full bass of the track. 

And if someone still wants to enable the equalizer with their preferred preset, it is only a click away.

~Raza
